easybuild/easyblocks/c/cuda.py

```diff
@@ -145,8 +146,8 @@ def make_module_req_guess(self):
 
         guesses.update({
             'PATH': ['open64/bin', 'bin'],
-            'LD_LIBRARY_PATH': ['lib64'],
-            'CPATH': ['include'],
+            'LD_LIBRARY_PATH': ['lib64', 'extras/CUPTI/lib64'],
```

Comment:
Are you sure it's `lib64`? If so, don't use `chk_libdir` above in the sanity check, but check for `"extras/CUPTI/lib64/libcupti.%s" % shlib_ext` instead.

Reply:
From the original cuda.py
"# Versions higher than 6 do not provide 32 bit libraries"

So chk_libdir uses in the sanity check is technically correct but if cupti doesn't exist prior to v7 then it can be skipped.

Reply:
Even in recent version only `extras/CUPTI/lib64` is there, there never is a `extras/CUPTI/lib` in my installations, so it doesn't make sense to use `chk_libdir` for the `extras/CUPTI/lib*` locations?

Reply:
Yeah, fixed that. But by main question is if the LD_LIBRARY_PATH should be dealt with to have 'lib' if version < 6?

Reply:
There is no `CUPTI` directory anywhere in CUDA5.x or 6.x, as far as I can tell.

Reply:
I was talking about "lib" and "lib64" in LD_LIBRARY_PATH. I.e. should "lib" be added if version < 6?
(Nothing to do with cupti here)

Reply:
We could consider including `lib` in case 32-bit builds are being done (we have a toolchain option for that), but including both `lib` and `lib64` is not a good idea imho...

Reply:
Ok. Skipping that part then.